### PR TITLE
Add CHANGELOG for v5.28.0

### PR DESCRIPTION
**Pull request by SarahFrench**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,48 @@
...
+* monitoring: added `service_agent_authentication` field to `google_monitoring_uptime_check_config` resource ([#17929](https://github.com/hashicorp/terraform-provider-google/pull/17929))
+* privateca: added `subject_key_id` field to `google_privateca_certificate` and `google_privateca_certificate_authority` resources ([#17923](https://github.com/hashicorp/terraform-provider-google/pull/17923))
+* secretmanager: added `version_destroy_ttl` field to `google_secret_manager_secret` resource ([#17888](https://github.com/hashicorp/terraform-provider-google/pull/17888))
+
+BUG FIXES:
+* appengine: added suppression for a diff in `google_app_engine_standard_app_version.automatic_scaling` when the block is unset in configuration ([#17905](https://github.com/hashicorp/terraform-provider-google/pull/17905))
+* sql: fixed issues with updating the `enable_google_ml_integration` field in `google_sql_database_instance` resource ([#17878](https://github.com/hashicorp/terraform-provider-google/pull/17878))
```
Comment (SarahFrench) [inline CHANGELOG.md:45]: Sorry about this - I've updated main with the 5.27.0 release notes, but after the release branch was created

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,48 @@
...
+* integrations: deprecated `create_sample_workflows` and `provision_gmek` fields in `google_integrations_client`.  ([#17945](https://github.com/hashicorp/terraform-provider-google/pull/17945))
+
+FEATURES:
+* **New Data Source:** `google_storage_buckets` ([#17960](https://github.com/hashicorp/terraform-provider-google/pull/17960))
+* **New Resource:** `google_compute_security_policy_rule` ([#17937](https://github.com/hashicorp/terraform-provider-google/pull/17937))
+* **New Resource:** `google_privileged_access_manager_entitlement` (beta) ([#17940](https://github.com/hashicorp/terraform-provider-google/pull/17940))
+
+IMPROVEMENTS:
+* alloydb: added `maintenance_update_policy` field to `google_alloydb_cluster` resource ([#17954](https://github.com/hashicorp/terraform-provider-google/pull/17954))
+* bigquery: promoted `external_dataset_reference` in `bigquery_dataset` to GA ([#17944](https://github.com/hashicorp/terraform-provider-google/pull/17944))
```
Comment (shuyama1) [inline CHANGELOG.md:13]:
```suggestion
* bigquery: promoted `external_dataset_reference` in `google_bigquery_dataset` to GA ([#17944](https://github.com/hashicorp/terraform-provider-google/pull/17944))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,48 @@
...
+* composer: promoted `config.0.software_config.0.image_version` in-place updates to GA in resource `google_composer_environment` ([#17986](https://github.com/hashicorp/terraform-provider-google/pull/17986))
```
Comment (shuyama1) [inline CHANGELOG.md:14]:
```suggestion
* composer: promoted `config.software_config.image_version` in-place update to GA in resource `google_composer_environment` ([#17986](https://github.com/hashicorp/terraform-provider-google/pull/17986))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,4 +1,48 @@
-## 5.27.0 (Unreleased)
+## 5.28.0 (Unreleased)
+
+DEPRECATIONS:
+* integrations: deprecated `create_sample_workflows` and `provision_gmek` fields in `google_integrations_client`.  ([#17945](https://github.com/hashicorp/terraform-provider-google/pull/17945))
+
+FEATURES:
+* **New Data Source:** `google_storage_buckets` ([#17960](https://github.com/hashicorp/terraform-provider-google/pull/17960))
+* **New Resource:** `google_compute_security_policy_rule` ([#17937](https://github.com/hashicorp/terraform-provider-google/pull/17937))
+* **New Resource:** `google_privileged_access_manager_entitlement` (beta) ([#17940](https://github.com/hashicorp/terraform-provider-google/pull/17940))
```
Comment (melinath) [inline CHANGELOG.md:9]:
```suggestion
```

beta-only

**Review by melinath — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (SarahFrench) [reply CHANGELOG.md:13]: Thanks!

**Pull request merged**